burr/cli/__main__.py

```diff
@@ -134,8 +134,10 @@ def _run_server(
     cmd = f"uvicorn burr.tracking.server.run:app --port {port} --host {host}"
     if dev_mode:
         cmd += " --reload"
-    base_dir = os.path.expanduser("~/.burr")
-    if not no_copy_demo_data:
+
+    if backend == "local" and not no_copy_demo_data:
```

Comment:
Best to instantiate the settings module if we can -- that way it's more future-proof/solves your TODO